en/api/events.md

```diff
@@ -235,6 +235,59 @@ Event emitted after range is changed in visualMap.
 }
 ```
 
+## graphroam(Event)
+Event emitted after [series-graph](option.html#series-graph) is roamed.

```

Comment:
There seem to be some redundant commits, please rebase to the latest master branch.